src/utils/spawnAsync.ts

```diff
@@ -240,3 +244,26 @@ export function bufferToString(buffer: Buffer): string {
     // eslint-disable-next-line no-control-regex
     return buffer.toString().replace(/[\x00-\x09\x0B-\x0C\x0E-\x1F]|\r?\n$/g, '');
 }
+
+function fixPathForMacIfNeeded(options: cp.SpawnOptions): void {
+    if (!isMac()) {
+        // Do nothing: not Mac
+        return;
+    }
+
+    // Looks for `/usr/local/bin` in the PATH.
+    // Must be whole, i.e. the left side must be the beginning of the string or :, and the right side must be the end of the string or :
+    // Case-insensitive, because Mac is
+    if (/(?<=^|:)\/usr\/local\/bin(?=$|:)/i.test(options?.env?.PATH)) {
+        // Do nothing: PATH already contains `/usr/local/bin`
+        return;
+    }
+
+    options = options ?? {};
+    options.env = options.env ?? process.env;

```

Comment:
```suggestion
    options.env = options.env ?? { ...process.env };
```

Might need to be this, to make a clone of `process.env` so that it does not get modified all-up.

Reply:
Good idea